src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -30,35 +31,58 @@ using namespace Windows::UI::Xaml::Media;
 using namespace Windows::UI::Xaml::Media::Imaging;
 using namespace Windows::UI::Popups;
 
-
 constexpr auto sc_ViewModelPropertyName = L"ViewModel";
 
-
 GraphingCalculator::GraphingCalculator()
 {
     Equation::RegisterDependencyProperties();
     Grapher::RegisterDependencyProperties();
     InitializeComponent();
 
-    DataTransferManager^ dataTransferManager = DataTransferManager::GetForCurrentView();
+    DataTransferManager ^ dataTransferManager = DataTransferManager::GetForCurrentView();
 
     // Register the current control as a share source.
-    m_dataRequestedToken = dataTransferManager->DataRequested += ref new TypedEventHandler<DataTransferManager^, DataRequestedEventArgs^>(this, &GraphingCalculator::OnDataRequested);
+    m_dataRequestedToken = dataTransferManager->DataRequested +=
+        ref new TypedEventHandler<DataTransferManager ^, DataRequestedEventArgs ^>(this, &GraphingCalculator::OnDataRequested);
+
+    // Request notifications when we should be showing the trace values
+    GraphingControl->TracingChangedEvent += ref new TracingChangedEventHandler(this, &GraphingCalculator::OnShowTracePopupChanged);
+
+    // And when the actual trace value changes
+    GraphingControl->TracingValueChangedEvent += ref new TracingValueChangedEventHandler(this, &GraphingCalculator::OnTracePointChanged);
 }
 
-void GraphingCalculator::GraphingCalculator_DataContextChanged(FrameworkElement^ sender, DataContextChangedEventArgs^ args)
+void GraphingCalculator::OnShowTracePopupChanged(bool newValue)
 {
-    ViewModel = dynamic_cast<GraphingCalculatorViewModel^>(args->NewValue);
+    TraceValuePopup->IsOpen = newValue;
+    if (TraceValuePopup->IsOpen)
+    {
+        // Set the keyboard focus to the graph control so we can use the arrow keys safely.
+        GraphingControl->Focus(::FocusState::Programmatic);
+    }
+}
+
+void GraphingCalculator::GraphingCalculator_DataContextChanged(FrameworkElement ^ sender, DataContextChangedEventArgs ^ args)
+{
+    ViewModel = dynamic_cast<GraphingCalculatorViewModel ^>(args->NewValue);
 
     ViewModel->VariableUpdated += ref new EventHandler<VariableChangedEventArgs>(this, &CalculatorApp::GraphingCalculator::OnVariableChanged);
 }
 
-GraphingCalculatorViewModel^ GraphingCalculator::ViewModel::get()
+void GraphingCalculator::OnTracePointChanged(Windows::Foundation::Point newPoint)
+{
+    TraceValuePopupTransform->X = (int)GraphingControl->TraceLocation.X + 15;
+    TraceValuePopupTransform->Y = (int)GraphingControl->TraceLocation.Y - 30;
+
+    TraceValue->Text = "(" + newPoint.X.ToString() + ", " + newPoint.Y.ToString() + ")";
```

Comment:
I think we want this entire string to be in the resw file, then pass the x/y string values as parameters into that string. You can see an example of this with the resw file entry for "Format_ValueFrom"

Reply:
Going to submit like this while PM talks to global readiness and figures out how (if) this should be handled.